### ISSUE TITLE
Thickened fetal skin 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7424]
**Preferred term label:**
thickened fetal skin 

See Figure 1 here (https://api.imrpress.com/sci/redirect/article/pdf?journal=CEOG&volumeCode=47&issueCode=6&doi=10.31083/j.ceog.2020.06.2096)

Is this a synonym of [Fetal skin edema HP:0025672](https://hpo.jax.org/app/browse/term/HP:0025672) ?

**Comment by sparkste** [comment]
Here are my thoughts:
Skin edema = fluid accumulation in the skin anywhere in the fetal body at any point in gestation
Thickened fetal skin = this sounds to me like it could be either skin edema or more like lymphangiectasia (which has a more heterogeneous but still thick appearance of the skin and is thought to result from aberrant lymphatics)
Thickened nuchal fold = thickened skin at the back of the fetal neck in the second trimester
Thickened/increased nuchal translucency = increased space at the back of the fetal neck in the first trimester
Let me know if I can provide more info!

**Comment by pnrobinson** [comment]
Looking back at the paper, the thickened skin is determined to be edematous in another part of the paper. Probably let's hold off for now with a new term, I think we have the components described above, thanks!
